metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/tags/ClassBasedTransformer.java

```diff
@@ -0,0 +1,36 @@
+package com.izettle.metrics.influxdb.tags;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Extract the class name and function from the metricName.
+ */
+public class ClassBasedTransformer implements Transformer{
```

Comment:
Formatting.

Reply:
So I add a space here and we add/fix checkstyle in another PR?